ArcGISRuntimeSDKQt_CppSamples/Scenes/SyncMapViewSceneView/SyncMapViewSceneView.json

```diff
@@ -0,0 +1,8 @@
+{
+    "source" :  "SyncMapViewSceneView.qml",
+    "otherSourceFiles" :  [
+        "SyncMapViewSceneView.h",
+        "SyncMapViewSceneView.cpp"
+    ],
+    "name" : "Sync MapView SceneView"
```

Comment:
Might want to update the title to match common design. This is currently being debated. Please work with Quincy to sort it out. FWIW, I'm liking `Sync map and scene view`